### ISSUE TITLE
Update docker images

### ISSUE THREAD

**Issue by GeoffClements** [rust-embedded/cross#209]
Noting #85, the docker image for armv7-unknown-linux-gnueabihf is based on ubuntu:14.04 which is okay providing you don't need to link to more modern libraries.

In my case I use the gstreamer crate which requires gstreamer-1.0 which is not available in ubuntu:14.04.

Please can you update to ubuntu:18.04.

**Comment by Dylan-DPC** [comment]
Hi thanks for reporting this.

However do check this [comment](https://github.com/rust-embedded/cross/pull/164#issuecomment-352188362) that explains why we haven't upgraded yet

**Comment by GeoffClements** [comment]
@Dylan-DPC Ok, thanks for the link.

That's means I will probably have to roll my own docker image based on Ubuntu or Debian late enough to support gstreamer1.0. I am aiming to run on RPi2/3 with [Tiny Core Linux](http://tinycorelinux.net/) which currently uses glibc 2.26 so it looks possible.

I have already tried a docker image based on Ubuntu:18.04 for the armhf architecture. In this case cross is not needed because I planned to run the arm versions of cargo and rustc through binfmt services on the host. It didn't work, I got many compile errors.
